src/ORM/DataQuery.php

```diff
@@ -660,9 +660,9 @@ public function having($having)
      *
      * @return DataQuery_SubGroup
      */
-    public function disjunctiveGroup()
+    public function disjunctiveGroup(string $clause = 'WHERE')
```

Comment:
This is an API breaking change. I know we've done this sort of thing in the past, but that was when there was no major releases on the horizon - and it _did_ cause problems even with our own modules.

Reply:
So non-BC approach would be to use `func_get_args()` and do a bit more logic within the method instead to mimic the default value?

Reply:
That would definitely work, yeah. I'd be okay with that.

Reply:
Have changed to `func_get_args`

Rather than creating a new card to add new params for CMS 6, which I think is highly likely to get lost and never done, I've instead added a `@deprecated` which is likely to get done since when we did CMS 5 we search for `@deprecated` and updated the things.  The alternative inline comment is `TODO` but these are almost certainly going to get ignored.